src/Microsoft.AspNet.Razor/CodeGenerators/Visitors/CSharpUsingVisitor.cs

```diff
@@ -11,7 +11,11 @@ namespace Microsoft.AspNet.Razor.CodeGenerators.Visitors
 {
     public class CSharpUsingVisitor : CodeVisitor<CSharpCodeWriter>
     {
-        private const string TagHelpersRuntimeNamespace = "Microsoft.AspNet.Razor.Runtime.TagHelpers";
+        private static readonly string[] TagHelpersRuntimeNamespaces = new[]
+        {
+            "Microsoft.AspNet.Razor.TagHelpers",
+            "Microsoft.AspNet.Razor.Runtime.TagHelpers.Internal"
```

Comment:
Why does the generated code need the `Internal` namespace? Does this make the types show up in IntelliSense when the user is in C# mode? I expected more of the explicit type qualification used elsewhere in the generated code.